pages/common/doctl-apps.md

```diff
@@ -0,0 +1,36 @@
+# doctl apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/apps>.
+
+- Create an app:
+
+`doctl apps create`
+
+- Create a deployment:
+
+`doctl apps create-deployment`
+
+- Delete an app interactively:
+
+`doctl apps delete {{app_id}}`
+
+- Get an app:
+
+`doctl apps get`
+
+- List all apps:
+
+`doctl apps list`
+
+- List all deployments:
+
+`doctl apps list-deployments`
+
+- Get logs:
+
+`doctl apps logs`
+
+- Update an app:
+
+`doctl apps update`
```

Comment:
The docs says the `--spec` argument is required. https://docs.digitalocean.com/reference/doctl/reference/apps/update/
```suggestion
`doctl apps update {{app_id}} --spec {{path/to/spec.yml}}`
```